hadoop-tools/hadoop-azure/src/site/markdown/abfs.md

```diff
@@ -812,6 +812,13 @@ aheads. Specify the value in bytes. The value should be between 16384 to
 104857600 both inclusive (16 KB to 100 MB). The default value will be
 4194304 (4 MB).
 
+`fs.azure.buffered.pread.disable`: By default the positional read API will do a
```

Comment:
We can clearly mentioned here that, it is not a property and it's used as a option in FutureDataInputStreamBuilder.

Reply:
Thanks.. Done.